fabric-data-generation-api-v1/src/main/java/net/fabricmc/fabric/api/datagen/v1/DataGeneratorEntrypoint.java

```diff
@@ -31,4 +33,15 @@ public interface DataGeneratorEntrypoint {
 	 * @param fabricDataGenerator The {@link FabricDataGenerator} instance
 	 */
 	void onInitializeDataGenerator(FabricDataGenerator fabricDataGenerator);
+
+	/**
+	 * Return the modid of the mod the data is being generated for.
+	 * A null return will run the data generation using the modid that registered the current entrypoint.
```

Comment:
We should document the behavior for nonexistent mod IDs (right now they crash with `RuntimeException`.)